database/migrations/2021_01_11_143724_create_patients_table.php

```diff
@@ -21,7 +21,7 @@ public function up()
             $table->string('phone')->unique();
             $table->string('photo');
             $table->integer('gender');
-            $table->timestamp('verified_at');
+            $table->timestamp('verified_at')->nullable();
```

Comment:
When editing an old migration, make a new migration for the edit

Reply:
i will take it in my account after that